web-common/src/features/dashboards/url-state/DashboardURLStateSyncWrapper.svelte

```diff
@@ -61,7 +61,7 @@
   $: parseUrl($page.url, defaultExplorePreset);
 </script>
 
-{#if !$validSpecStore.isLoading && !$metricsViewTimeRange.isLoading}
+{#if !$validSpecStore.isLoading}
```

Comment:
We probably still need `metricsViewTimeRange.isLoading` when the dashboard has a timestamp. Maybe the reason E2E is failing.